larray/io/excel.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, filepath=None, overwrite_file=False, visible=None, silent=Non
 
             xw_wkb = None
             self.delayed_filepath = None
+            self.filepath = None
```

Comment:
I wonder if we shouldn't have only one filepath. Ie reuse delayed_filepath in that case?